sdk/iot/Azure.Iot.Hub.Service/src/JobsClient.cs

```diff
@@ -3,23 +3,205 @@
+
+            var jobProperties = new JobProperties
+            {
+                Type = JobPropertiesType.Import,
+                InputBlobContainerUri = importBlobContainerUri.ToString(),
+                OutputBlobContainerUri = outputBlobContainerUri.ToString(),
+                StorageAuthenticationType = options?.AuthenticationType,
+                InputBlobName = options?.InputBlobName,
+                OutputBlobName = options?.OutputBlobName,
+            };
+
+            return _jobsRestClient.CreateImportExportJobAsync(jobProperties, cancellationToken);
+        }
+
+        /// <summary>
+        /// List all import and export jobs for the IoT Hub.
+        /// </summary>
+        /// <param name="cancellationToken">Task cancellation token.</param>
+        /// <returns>Collection of JobProperties of all jobs for this IoT Hub.</returns>
+        public virtual Response<IReadOnlyList<JobProperties>> GetImportExportJobs(CancellationToken cancellationToken = default)
+        {
+            return _jobsRestClient.GetImportExportJobs(cancellationToken);
+        }
+
+        /// <summary>
+        /// List all import and export jobs for the IoT Hub.
+        /// </summary>
+        /// <param name="cancellationToken">Task cancellation token.</param>
+        /// <returns>Collection of JobProperties of all jobs for this IoT Hub.</returns>
+        public virtual Task<Response<IReadOnlyList<JobProperties>>> GetImportExportJobsAsync(CancellationToken cancellationToken = default)
+        {
+            return _jobsRestClient.GetImportExportJobsAsync(cancellationToken);
+        }
+
+        /// <summary>
+        /// Gets the import or export job with the specified ID.
+        /// </summary>
+        /// <param name="jobId">Id of the Job to retrieve.</param>
+        /// <param name="cancellationToken">Task cancellation token.</param>
+        /// <returns>JobProperties of the job.</returns>
+        public virtual Response<JobProperties> GetImportExportJob(string jobId, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(jobId, nameof(jobId));
+
+            return _jobsRestClient.GetImportExportJob(jobId, cancellationToken);
+        }
+
+        /// <summary>
+        /// Gets the import or export job with the specified ID.
+        /// </summary>
+        /// <param name="jobId">Id of the Job to retrieve.</param>
+        /// <param name="cancellationToken">Task cancellation token.</param>
+        /// <returns>JobProperties of the job.</returns>
+        public virtual Task<Response<JobProperties>> GetImportExportJobAsync(string jobId, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(jobId, nameof(jobId));
+
+            return _jobsRestClient.GetImportExportJobAsync(jobId, cancellationToken);
+        }
+
         /// <summary>
-        /// place holder
+        /// Cancels or deletes the job with the specified ID.
```

Comment:
Pretty sure we've been using "Id" everywhere, instead of "ID".

Reply:
e.g. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs#L113

Reply:
Good catch.